app/javascript/vue/protocol/modals/publish_protocol.vue

```diff
@@ -5,7 +5,7 @@
         <div class="modal-header">
           <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
           <h4 class="modal-title">
-            {{ i18n.t('protocols.publish_modal.title')}}
+            {{ i18n.t('protocols.publish_modal.title', { nr: protocol.attributes.version + 1 })}}
```

Comment:
Why this + 1

Reply:
@aignatov-bio 
`protocol.attributes.version` is the version of the published protocol that the draft is based on. So in the title of the publish modal, we want to display the version of the protocol if it were to be published. Hence the `+ 1`.